src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTransformer.java

```diff
@@ -292,4 +464,67 @@ private void addApiImports(GapicInterfaceContext context) {
 
     return apiMethods;
   }
+
+  private String generateConfigOutputPath(GapicInterfaceContext context, String name) {
+    GapicInterfaceConfig interfaceConfig = context.getInterfaceConfig();
+    String outputPath =
+        pathMapper.getOutputPath(
+            context.getInterfaceModel().getFullName(), context.getProductConfig());
+    return outputPath.substring(0, outputPath.length() - 6)
```

Comment:
As discussed, lets restructure this.

Reply:
On it!

How does `public String getConfigPath(InterfaceConfig interfaceConfig, String name)` in SurfaceNamer sound?

I tossed around the idea of refactoring `getClientConfigPath` into the more generic `getConfigPath`, but it seems rather disruptive, what about just having these two methods live next to each other?

Reply:
SGTM